### PR TITLE
Remove deprecated `CupertinoContextMenu.previewBuilder`

### PR DESCRIPTION
**Pull request by LongCatIsLooong**
It was deprecated in https://github.com/flutter/flutter/pull/110616.

Deprecated in https://github.com/flutter/flutter/pull/110616

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[Data Driven Fixes]: https://github.com/flutter/flutter/wiki/Data-driven-Fixes

### CODE REVIEW

**Review by nate-thegrate — COMMENTED** [review]
I took a look at the failing tests—looks like all that needs to be done is to remove `[previewBuilder]` from the context_menu.dart doc comments.

```dart
/// The [previewBuilder] parameter is most commonly used to display...
```

File: packages/flutter/lib/src/cupertino/context_menu.dart [hunk 1]
```diff
@@ -56,15 +56,6 @@ typedef _DismissCallback = void Function(
   double opacity,
 );
 
-/// A function that produces the preview when the CupertinoContextMenu is open.
-///
-/// Called every time the animation value changes.
-typedef ContextMenuPreviewBuilder = Widget Function(
```
Comment (LongCatIsLooong) [inline packages/flutter/lib/src/cupertino/context_menu.dart:62]: This wasn't deprecated in the deprecation PR but it doesn't seem to be used anywhere else.

**Review by MitchellGoodwin — APPROVED** [review]
LGTM

**Review by Piinks — APPROVED** [review]
LGTM

**Comment by matanlurey** [comment]
test-exempt: non-semantic refactor.

**Comment by sudhar08** [comment]
Hey guys the preivewbuilder is an awsome feture why that was been removed i kindly resquest you to add in the feature upcoming upgrade from the flutter Thank you  !!!

**Comment by sudhar08** [comment]
Hi @LongCatIsLooong in the **Cupertiocontexmenu.builder**  implement the preview  can you please explain me how to do that

**Pull request merged**